### ISSUE TITLE
Regression: PCRE regex no longer supported

### ISSUE THREAD

**Issue by tripleee** [Charcoal-SE/metasmoke#258]
We used to be able to use some PCRE regex convenience operators like `\W` and `\b` in the interactive Metasmoke search. This one should get results, but currently doesn't:

https://metasmoke.erwaysoftware.com/search?body_is_regex=1&body=%5Cbtest%5Cb

The workaround is not too bad, you can use `[^A-Za-z0-9_]` for `\W` and something like `(^|[^A-Za-z0-9_])` for `\b` at beginning of line, and similarly for end of line:

https://metasmoke.erwaysoftware.com/search?body_is_regex=1&body=%28%5E%7C%5B%5EA-Za-z0-9_%5D%29test%28%5B%5EA-Za-z0-9_%5D%7C%24%29

(Currently returns 2310 results.)

I remember how this began to work somewhat to my surprise without any announcement from the developers at some point earlier this year (vaguely? or maybe before Christmas?), so I'm guessing it's a configuration option or something. As far as I can tell, it stopped working when Undo migrated Metasmoke to a new installation yesterday.

**Comment by tripleee** [comment]
It's unclear to me how this ever worked. https://stackoverflow.com/questions/14081138/use-of-pcre-regular-expressions-with-mysql says it's an optional plug-in (or you could switch to MariaDB, did someone do that for the old Metasmoke instance?)

**Comment by tripleee** [comment]
I now know more about things which were unclear when I submitted this ticket, though I can no longer point to when or where exactly I learned about these circumstances.  Metasmoke was briefly running on MariaDB for a while, but performance turned out to be worse than with MySQL, so it was switched back to MySQL.  My understanding is that this was an incidental switch-over when Metasmoke had to be migrated to a new host (a couple of years back IIRC?) but if somebody actually did performance measurements, maybe it was more disciplined than that after all.
